Fix error: 'NAME_MAX' undeclared (first use in this function)
--- a/cmds/filesystem-usage.c
+++ b/cmds/filesystem-usage.c
@@ -24,6 +24,7 @@
 #include <stdarg.h>
 #include <getopt.h>
 #include <fcntl.h>
+#include <limits.h>

 #include "common/utils.h"
 #include "kerncompat.h"
